SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -502,11 +477,12 @@
     </PropertyGroup>
 
     <ItemGroup Condition=" $(SonarLintFound) == 'true' ">
-      <!-- Remove any existing references to files called *SonarLint*.dll -->
-      <Analyzer Remove="@(Analyzer)" Condition="$([System.Text.RegularExpressions.Regex]::IsMatch($([System.String]::new('%(Analyzer.Identity)')), 'SonarLint[^\\]*\.dll$', System.Text.RegularExpressions.RegexOptions.IgnoreCase)) " />
-      <Analyzer Include="$(SonarLintAnalyzerDir)*SonarLint*.dll" />
-      
-      <!-- Add SonarLint.xml as a Roslyn Additional File so that SonarLint can pick up the right rule parameter values -->
+      <!-- Remove any existing analyzers and additional files -->
+      <Analyzer Remove="@(Analyzer)" />
```

Comment:
@dbolkensteyn Yes, it is. We aren't going to upload the issues, and we aren't using the ruleset configuration the user has specified for any additional analyzers, so we don't want to run the extra analysers.